.git-hooks/locale.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want 
+ * to block the upload of modified files in the "locale" 
+ * folder to the repository through a git hook generated 
+ * with the Husky library.
+ * 
+ * Usage: $ node .git-hook/locale.js
+ * 
+ * Happy releasing!
+ */
+
+'use strict'
+
+// Require main dependences.
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Execute Git Diff for get all files with changes.
+const results = execSync('git diff --cached --name-only')
+
+// Create array with all files.
+const arrayResults = results.toString().split('\n')
+// Remove last element with empty line.
+arrayResults.pop()
+
+// Define array files to not change.
+const filesNoChange = []
+
+function gitHook(expresion) {
+
+  // Define RegExp with reg param.
+  const reg = new RegExp(expresion, 'g')
```

Comment:
I think this would be more readable as `new RegExp('/locale/', 'g')` rather than making the expression an argument to the function.

Reply:
I think it's a good idea. Done!